smarts/core/smarts.py

```diff
@@ -875,7 +882,7 @@ def _try_emit_envision_state(self, provider_state, obs, scores):
                         agent_id, v.vehicle_id, is_multi=is_boid_agent,
                     ),
                     events=vehicle_obs.events,
-                    waypoint_paths=vehicle_obs.waypoint_paths or [],
+                    waypoint_paths=(vehicle_obs.waypoint_paths or []) + road_waypoints,
```

Comment:
Fine for now, but would look better if we differentiate if these are path waypoints vs all road waypoints.